source/middlewares.html

```diff
@@ -252,5 +252,37 @@
             </div>
         </div>
     </div>
+    <div class="row-fluid">
+        <div class="span6">
+            <h3>
+                <a href="https://github.com/thecodingmachine/silex-middleware">Silex middleware</a>
+            </h3>
+            <p class="by">
+                by moufmouf
+            </p>
+            <p>
+                Turns a Silex application into a Stack middleware.
+            </p>
+            <div class="btn-group">
+                <a class="btn" href="https://github.com/thecodingmachine/silex-middleware"><i class="icon icon-github"></i> GitHub</a>
+                <a class="btn" href="https://packagist.org/packages/mouf/silex-middleware"><i class="icon icon-archive"></i> Packagist</a>
+            </div>
+        </div>
+        <div class="span6">
+            <h3>
+                <a href="https://github.com/thecodingmachine/whoops-stackphp">Symfony 2 middleware</a>
```

Comment:
@mnapoli Hm, it works for me? Either @moufmouf fixed it or there was a github hiccup earlier?